en-US/win10/SetupTPM.md

```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
 ##<a name="dTPM"></a>Setup dTPM  
 These instructions are applicable for any dTPM module supported on either MBM or RPi2.
 
-1. Get a discrete TPM module and attach it to the MBM.
-2. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
-3. Enable testsigning on the device:
-
-        bcdedit /set {current} integrityservices disable
-        bcdedit /set testsigning on
-
-4. Copy the ACPI table to MBM _C:\Windows\System32\ACPITABL.dat_.
-5. Disable fTPM by changing the following UEFI settings:
+1. Get a discrete TPM module and attach it to the MBM/RPi2.
```

Comment:
No, I do not have a pointer to supported modules that is available publicly.
The handful of TPM modules that we verified on MBM and RPi2 are not (yet) retail sold (off-the-shelf). My understanding is that the makers/OEMs need to contact the TPM Manufacturers and arrange purchase directly. This is more of a Maker-pro scenario.
merzink should be able to confirm and provide more details here.